### ISSUE TITLE
Getting TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"

### ISSUE THREAD

**Issue by swati-patro** [ortoniKC/Playwright-Test-Runner#2]
While running "clip.test.ts" file, getting error like "TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts"" but other ts file working fine. Only having issue when i tried to run clip test case

**Comment by ortoniKC** [comment]
I believe u have installed the clipboardy package

**Comment by swati-patro** [comment]
yes installed,  using command "npm install clipboardy -D" but still I am getting this issue
